modules/ROOT/pages/managing-permissions.adoc

```diff
@@ -8,49 +8,49 @@ This section explains how permissions work across different products and APIs ma
 
 == Assumptions
 
-This page assumes that you have an Organization Administrator role in your organization, that you have been assigned as the administrator of one of the business groups of your Organization, or that you have API Version Owner permissions and that you want to manage user permissions for an API version, a business group or the entire organization.
+This page assumes that you have an Organization Administrator permission in your organization, that you have been assigned as the administrator of one of the business groups of your Organization, or that you have API Version Owner permissions and that you want to manage user permissions for an API version, a business group or the entire organization.
 
-== How Permissions Work in the Anypoint Platform
+== How Permissions Work in Anypoint Platform
 
-In the Anypoint Platform, users belong to an organization and have a set of *roles* and *permissions*.
+In the Anypoint Platform, users belong to an organization and have a set of permissions. These permissions are assigned using xref:teams.adoc[teams], xref:roles.adoc[roles] (deprecated), or by xref:users.adoc#grant-user-permissions[granting permissions to users individually].
 
-Each role contains a list of _permissions_ that define what a user that holds that role can do with the specific resources it scopes. +
-Certain Roles come with a set of default permissions. As an Organization Administrator, you can create your customized roles and assign the permissions you see fit, or, depending on the product, you can add permissions directly to a specific user, without the need for roles.
+The Teams feature enables you to create groups of users and then assign the members of those groups certain permissions. As an Organization Administrator, you can create and modify teams and assign permissions according to organizational structure.
```

Comment:
Can we mention that teams can have permissions from multiple business groups / inheritance semantics while roles do not? Right now it sounds like teams and roles do the same thing and it isn't clear why roles are deprecated

Reply:
```suggestion
The Teams feature enables you to create groups of users and then assign the members of those groups certain permissions across multiple business groups. As an Organization Administrator, you can create and modify teams and assign cascading permissions according to organizational structure.
```